### PR TITLE
import: use quote formatting verb (%q)

### PR DESCRIPTION
**Pull request by aliculPix4D**
- import: use quote formatting verb (%q)

Allows for the use of multi word addresses. i.e commonly encountered for label names

### CODE REVIEW

File: cmdimport.go [hunk 1]
```diff
@@ -30,9 +30,12 @@ type Definitions struct {
 
 // Keep track of the asymmetry of import subcommand.
 // When importing, the up direction wants two parameters:
-//   terraform import res-address res-id
+//
```
Comment (aliculPix4D) [inline cmdimport.go:33]: this is probably my editor doing the formatting. If you wish I can revert this change

File: testdata/import/08_import_up.sh [hunk 2]
```diff
@@ -2,35 +2,38 @@
 # DO NOT EDIT. Generated by terravalet.
 # WARNING: check the order of resources before running this script.
 #
-# This script will "terraform import" 9 items.
+# This script will "terraform import" 10 items.
 
 # Uncomment this if you want to stop the script at first error
 # set -e
 
 terraform import \
```
Comment (aliculPix4D) [inline testdata/import/08_import_up.sh:10]: I tested these changes in up script on 1 real repo by doing the full import cycle.

**Review by Esysc — APPROVED** [review]

File: cmdimport.go (hunk 1 above)
Comment (marco-m-pix4d) [reply cmdimport.go:33]: This should be `go fmt` (invoked by the editor); it changed in Go 1.19.
Could you try with `go fmt ./...` ?

For the difference between `gofmt` and `go fmt` (🤯 ) see https://www.youtube.com/watch?v=i8zjh-PW7bg

**Review by odormond — APPROVED** [review]
Let's hope that we cannot create a label named `$(rm -fr /)` in github...

File: cmdimport.go (hunk 1 above)
Comment (aliculPix4D) [reply cmdimport.go:33]: > Would it be possible to add a repro test, that fails with the current terravalet and passes with this PR ?

I am not sure how I would do that. Its not a problem of terravalet. Its how terraform would interpret:
```
terraform import \
    'module.github.github_branch_protection_v3.settings["test-import-gh:master"]' multi word address
```
compared to:
```
terraform import \
    'module.github.github_branch_protection_v3.settings["test-import-gh:master"]' 'multi word address'
```

File: testdata/import/08_import_src-plan.json [hunk 3]
```diff
@@ -134,6 +134,25 @@
             "name": "test-import-bar"
           }
         }
-      }
+      },
+      {
+        "address":"module.github.github_issue_label.all_labels[\"test-import-bar.multi word label\"]",
```
Comment (marco-m-pix4d) [inline testdata/import/08_import_src-plan.json:139]: Ah yes, I missed this. Thanks.

**Review by marco-m-pix4d — APPROVED** [review]

File: cmdimport.go (hunk 1 above)
Comment (marco-m-pix4d) [reply cmdimport.go:33]: I missed https://github.com/Pix4D/terravalet/pull/51#discussion_r1112706478

**Pull request merged**